upath/tests/implementations/test_local.py

```diff
@@ -15,3 +15,14 @@ def path(self, local_testdir):
 
     def test_is_LocalPath(self):
         assert isinstance(self.path, LocalPath)
+
+
+@skip_on_windows
```

Comment:
add `from upath.tests.utils import xfail_if_version` and

```suggestion
@skip_on_windows
@xfail_if_version("fsspec", lt="2023.10.0", reason="requires fsspec>=2023.10.0")
```